src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -32,9 +32,12 @@
      * The amount of items returned by certain crafting recipes
      */
     private static final int ONE_FORTH_OF_A_STACK = 16;
+
+    private static final int FOUR_BLOCKS = 4;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "FOUR_BLOCKS" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)